Env check

alidauda opened the pull request:
fix: improve error handling for missing WEBHOOK_SECRET in signature verification and the mono client

alidauda committed:
fix: improve error handling for missing WEBHOOK_SECRET in signature verification

sin4ch commented:
@alidauda merge conflicts here too.

I made some type and style refactors to the code. Will appreciate it if you could resolve the merge conflicts because you should be good to go.

sin4ch committed:
Merge branch 'main' into ENV-CHECK

sin4ch committed:
Raise ValueError for unset WEBHOOK_SECRET

sin4ch committed:
Update webhook signature verification logic

Modify webhook signature verification to return False when WEBHOOK_SECRET is not set.
